crates/hdk/README.md

```diff
@@ -101,16 +76,16 @@ debug!(links);
 let agent_pubkey = agent_info()?.agent_pubkey;
 // Get the name of this zome
 let zome_name = zome_info()?.zome_name;
-// Call your friends foo function
+// Call your friends "foo" function
 let result: SerializedBytes = call_remote(
     my_friends_agent_pubkey,
     zome_name,
     "foo".to_string(),
     CapSecret::default(),
-    MyInput.try_into()?
+    MyInput
```

Comment:
MyInput just needs to be serializable here, not serialized, so that's why it's fine to pass this